entity-types/ext-cacti/definition.yml

```diff
@@ -0,0 +1,21 @@
+domain: EXT
+type: CACTI
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: PingLatencySample
+    - attribute: metricName
+      value: "value"
```

Comment:
Is this really the desired value?

Reply:
Yes @manuelrodriguez-nr, the data is received within the `value` attribute